app/services/lazykhschduler.py

```diff
@@ -0,0 +1,159 @@
+import argparse
+import os.path
+import json
+import numpy as np
+import random
+
+def addPhoneme(p, t):
+    global prevPhoneme
+    global f
+    if p != prevPhoneme:
+        strings[4] += (str.format('{0:.3f}', t)+",phoneme,"+p+"\n")
+    prevPhoneme = p
+
+def pickNewPose(t):
+    global pose
+    global prevPose
```

Comment:
Use the naming convention for Python, see https://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function